components/lib/crash/src/main/java/mozilla/components/lib/crash/service/GleanCrashReporterService.kt

```diff
@@ -36,6 +36,12 @@ class GleanCrashReporterService(
         // as the persisted crashes in the crash count file (see above comment)
         const val UNCAUGHT_EXCEPTION_KEY = "uncaught_exception"
         const val CAUGHT_EXCEPTION_KEY = "caught_exception"
+        const val MAIN_PROCESS_NATIVE_CODE_CRASH_KEY = "main_proc_native_code_crash"
+        const val FOREGROUND_CHILD_PROCESS_NATIVE_CODE_CRASH_KEY = "fg_proc_native_code_crash"
```

Comment:
nit: for me just using foreground / background (vs fg / bg) here is more readable

Reply:
I think `foreground_native_code_crash` (without the `proc`) would fit. Would you prefer that?

Reply:
Sure! That sounds good too.

Reply:
Ah, no. Unfortunately none of those options work. I think we'll need to leave it as it is...

Reply:
No worries, just a small nit.  Thanks